### DISCUSSION TITLE
🐛 [BUG] renderer为svg时，图例legend分页点击上一页无反应

### DISCUSSION THREAD

**Discussion by ufresh2013** [antvis/G2Plot#3570] [Q&A] [answered]
### 🐛 Bug description
renderer为svg时，图例legend分页点击上一页无反应

### 📷 Step to reproduce
```
import { Column } from '@antv/g2plot';

fetch('https://gw.alipayobjects.com/os/antfincdn/P14mCvkybz/large-datra.json')
  .then((data) => data.json())
  .then((data) => {
    const plot = new Column('container', {
      data,
      xField: 'product_box',
      renderer:'svg',
      yField: 'value',
      seriesField: 'province',
      isGroup: 'true',
    });

    plot.render();
  });
```

### 🏞 Expected result
图例分页点击上一页能正常返回上一页，且缩放窗口大小，不会导致图例错位

### 🚑 Any additional
* **G2Plot Version**: 2.4.25
* **Platform**: Chrome 111.0.5563.65（正式版本） （64 位）

**Comment by hustcc** [comment, marked as answer]
暂时不要使用 SVG renderer 模式吧，当前确实问题挺多，很多是架构设计问题，难以解决。关于 SVG 渲染，我们在 G2 5.0 新版本中会进行严格测试，稳定性会高很多。 

如果确定一定需要用 SVG，建议使用 G2 5.0。
